bindings/py/setup.py

```diff
@@ -561,7 +561,7 @@ def getExtensionModules(nupicCoreReleaseDir, platform, bitness, cxxCompiler, cmd
     setup(
       name="nupic.bindings",
       ext_modules=extensions,
-      version="0.2.1",
+      version="0.2.2",
```

Comment:
bindings 0.2.2 have already been released to PyPi. You will need to bump the version